data/_ui/interfaces.txt

```diff
@@ -11,6 +11,10 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+# Raw color terms
+color black 0. 0. 0. 1.
+color red 1. 0. 0. 1.
```

Comment:
These colors should be renamed and moved down to be used only for the error text (maybe something like "map route error" and "map route error shadow"). This will allow people to change the color to, for example, orange without changing a global "red" color to orange.

Reply:
It is intentional, this is how CSS frameworks usually work; there are two levels of indirection. It is used by declaring at the top raw colours with appropriate names, and then farther down the file a colour-use-name like "primary warning" gets defined as e.g. "bright red". And a shared concept of "bright red" means that use bases other than warnings can share the same scheme.

Reply:
Fair enough. I haven't played this game for a couple of months as I've been quite busy evenings and weekends (mostly cleaning up after 5 kids). Please use anything you want to, and discard anything you don't.